### DISCUSSION TITLE
Initial conditions for 2000-?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [GEOS-ESM/GOCART#195] [Q&A]
Hello, 

I have been using GOCART as part of the development of the coupled UFS application, which is currently focused on the period from 2011-2018. I was wondering if aerosol initial conditions exist for a broader time range, ideally back to 2000. If so, where would I find them? If not does anyone have any suggestions for what to do (initialize with climatology, interpolate a different data source, ?). Thanks!

**Comment by mathomp4** [comment]
@benjamin-cash This should be doable, yes. GEOS using GOCART does support GOCART before 2000 with AMIP.20C a la here:

https://github.com/GEOS-ESM/GOCART/tree/main/ESMF/GOCART2G_GridComp/CA2G_GridComp/AMIP.20C

Unfortunately, using these requires the use of a "magic date" if you are not using ExtData2G, see https://github.com/GEOS-ESM/GEOSgcm_App/blob/6a3892f3f6e28f0cfebf5a7d8963e3c3cf03d618/gcm_run.j#L585

So in GEOS, we use AMIP.20C until we get to 2000-03-01, then we transition to AMIP. I think this is mainly due to HFED vs. QFED, though I'm not sure? And I'm not sure when the earliest date we support is, but emissions from 2000 onward should be easily doable.

I'll ping @amdasilva and he can probably speak to this better than I can.

**Comment by bbakernoaa** [comment]
For our UFS-Aerosol testing in the prototype simulations we start from a MERRA2 IC.  The fire emissions as @mathomp4 suggests is the key for the retrospective.  I have already copied the HFED emissions for pre-QFED dates.
